cpp/src/slice2rb/RubyUtil.cpp

```diff
@@ -875,15 +775,11 @@ Slice::Ruby::CodeVisitor::visitEnum(const EnumPtr& p)
     //
     // from_int
     //
-    {
-        _out << sp << nl << "def " << name << ".from_int(val)";
-        ostringstream sz;
-        sz << enumerators.size() - 1;
```

Comment:
No idea what this `ostringstream` was for.
Or why this code is in it's own block.
Seemed pointless though.